src/utils.ts

```diff
@@ -30,11 +38,14 @@ export const runIframe = (authorizeUrl: string, eventOrigin: string) => {
     const iframeEventHandler = function(e: MessageEvent) {
       if (e.origin != eventOrigin) return;
       if (!e.data || e.data.type !== 'authorization_response') return;
-      (<any>e.source).close();
```

Comment:
It doesn't. Since we're sure this is an iframe, `close` in this case would just remove the iframe from the DOM, which we're doing manually a few lines below